internal/pubsub/rabbitpubsub/rabbit.go

```diff
@@ -184,6 +199,27 @@ func (t *topic) receiveFromPublishChannels(ctx context.Context, nMessages int) e
 	return err
 }
 
+// Return the error from a Go channel monitoring the closing of an AMQP channel.
+// closec must have been registered via Channel.NotifyClose.
+// When closeErr is called, we expect closec to be closed. If it isn't, we also
+// consider that an error.
+func closeErr(closec <-chan *amqp.Error) error {
+	select {
+	case aerr := <-closec:
```

Comment:
Why not just `err`?

Reply:
To make it a little more clear that it's not of type `error`.

Reply:
Oh, I see, it only satisfies `error`. Okay.